internal/endian_be.go

```diff
@@ -1,4 +1,4 @@
-//go:build armbe || arm64be || mips || mips64 || mips64p32 || ppc64 || s390 || s390x || sparc || sparc64
```

Comment:
Looking at https://cs.opensource.google/go/x/sys/+/refs/tags/v0.18.0:cpu/byteorder.go;l=44, it seems loong64 is actually a little endian arch